Apps/frontend/src/views/Waves.svelte

```diff
@@ -48,10 +48,25 @@
     }
   };
 
+  // Sets the scaling of a individual wave according to the voltage intervals
+  const setScaling = (index, scale) => {
+    lines[index].scaleY = computeScaling(scale);
+  };
+
+  // computes the Scaling of a wave according to the voltage intervals
+  const computeScaling = (scale) => {
+    return (1 / (NUM_INTERVALS_Y / 2)) * scale;
+  };
+
   export const updateChannelOffsetY = (channelIndex, offsetY) => {
     lines[channelIndex].offsetY = offsetY;
   };
 
+  // Update the amplification of wave
+  export const updateChannelAmplitude = (channelIndex, amplitude) => {
+    setScaling(channelIndex, amplitude);
```

Comment:
I would remove setScaling and insert the one-liner right here

Reply:
I did this on purpose, because it makes refactoring easier, in case we want to make the scaling globally changeable in addition. I also don't see the advantage why this should not be encapsulated in a separate function.